docs/src/tutorials/conic/quantum_discrimination.jl

```diff
@@ -0,0 +1,126 @@
+# Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors    #src
+# This Source Code Form is subject to the terms of the Mozilla Public License   #src
+# v.2.0. If a copy of the MPL was not distributed with this file, You can       #src
+# obtain one at https://mozilla.org/MPL/2.0/.                                   #src
+
+# # Quantum state discrimination
+
+# This tutorial solves the problem of [quantum state discrimination](https://en.wikipedia.org/wiki/Quantum_state_discrimination).
+
+# The purpose of this tutorial to demonstrate how to solve problems involving
+# complex-valued decision variables and the [`HermitianPSDCone`(@ref). See
+# [Complex number support](@ref) for more details.
+
+# ## Required packages
+
+# This tutorial makes use of the following packages:
+
+using JuMP
+import LinearAlgebra
+import SCS
+
+# ## formulation
+
+# A `d`-dimensional quantum state, ``\rho``, can be defined by a complex-valued
+# Hermitian matrix with a trace of `1`. Assume we have `N` `d`-dimensional
+# quantum states, ``\{\rho_i}_{i=1}^n``, each of which is equally likely.
+
+# The goal of the Quantum state discrimination problem is to choose a set of
+# positive-operator-valued-measures (POVMs), ``E_i`` such that if we observe
+# ``E_i`` then the most probable state that we are in is ``\rho_i``.
+
+# Each POVM ``E_i`` is a complex-valued Hermitian matrix, and there is a
+# requirement that ``\sum\limits E_i = \mathbf{I}``.
```

Comment:
This is what I've inferred by reading https://arxiv.org/pdf/1707.02571.pdf.

But I don't actually understand what a POVM is in practice...

Reply:
And neither does vale by the look of it :)

Reply:
Fixed vale